shuup/admin/static_src/base/js/form-utils.js

```diff
@@ -56,8 +56,9 @@ window.renderFormErrors = function($form, errors) {
             if(fieldName === "__all__") {
                 $form.parent().find(".errors").append('<div class="alert alert-danger">' + fieldErrors + '</div>');
             } else {
-                let $field = $form.find(":input[name='" + formName + "-" + fieldName + "']").parent(".form-group");
-                $field.append("<span class='help-block error-block'>" + fieldErrors + "</span>").addClass("has-error");
+                var x = document.getElementsByName(formName + "-" + fieldName)[0];
```

Comment:
Check this solution:
```js
window.renderFormErrors = function($form, errors) {
    Object.keys(errors).forEach((formName) => {
        const formErrors = errors[formName];
        Object.keys(formErrors).forEach((fieldName) => {
            const fieldErrors = formErrors[fieldName].join(" ");
            if(fieldName === "__all__") {
                $form.parent().find(".errors").append('<div class="alert alert-danger">' + fieldErrors + '</div>');
            } else {
                const field = $("[name=" + formName + "-" + fieldName + "]");
                field.closest(".form-group").addClass("has-error");
                field.closest(".form-group").append("<span class='help-block error-block'>" + fieldErrors + "</span><br>");
            }
        });
    });
};
```

Reply:
Changed that up a bit to match the styles.